packages/react-scripts/scripts/utils/frontierInit.js

```diff
@@ -60,59 +57,59 @@ function installFrontierDependencies(appPath, answers, useYarn, ownPath) {
   ];
 
   const defaultDevModules = [
+    'eslint@5.6.0',
+    'fs-webdev/eslint-config-frontier#settingUpRecommendedReactConfig',
```

Comment:
start putting eslint-config-frontier into artifactory, and then update that usage here